tutorials/kube-mqtt/refresher-container/src/run_bridge.py

```diff
@@ -0,0 +1,78 @@
+"""Binary for running Mosquitto MQTT bridge with credential rotation."""
+import datetime
+import os
+import signal
+import subprocess
+import time
+import jwt
+import psutil
+
+
+def create_jwt_token():
+  """Creates a JWT (https://jwt.io) to establish an MQTT connection.
+
+  Returns:
+      An MQTT auth token generated from the given project_id and private
+      key
+  Raises:
+      ValueError: If the private_key_file does not contain a known key.
+  """
+
+  token = {
+      # The time that the token was issued at
+      'iat':
+          datetime.datetime.utcnow(),
+      # The time the token expires.
+      'exp':
+          datetime.datetime.utcnow() +
+          datetime.timedelta(minutes=int(os.environ["JWT_EXPIRATION_MINUTES"])),
+      # The audience field should always be set to the GCP project id.
+      'aud':
+          os.environ["PROJECT_ID"]
+  }
+
+  # Read the private key file.
+  with open("/etc/mqtt/ec_private.pem", 'r') as f:
+    private_key = f.read()
```

Comment:
Spacing looks a little inconsistent on this line relative to others.  Maybe lint with:

```
flake8 --show-source --builtin gettext --max-complexity 20 --import-order-style google --exclude ".nox,.cache,env,lib,generated_pb2,*_pb2.py,*_pb2_grpc.py" --ignore=E121,E123,E126,E226,E24,E704,W503,W504,I100,I201,I202 *.py
```

Reply:
I'm using default flake8 and pylint in vscode and wasn't seeing red-squiggles...  Is that flake8 our goog standard?

Reply:
Flake8 with our set of rules is how we lint our samples